kip-002.md

```diff
@@ -0,0 +1,52 @@
+# Kolibri Improvement Proposal #002
+
+Author: Keefer Taylor, Hover Labs <keefer@hover.engineering>"
+Created: 04/08/2021
+State: Draft
+
+### Summary
+
+Sets economic parameters in which we'd consider lowering the stability fee based on economic data.
+
+### Motivation
+
+Kolibri's `stability fee` (the rate charged to borrowers) is currently 22%. While this has helped keep peg stable, the high rate may be hindering the growth and adoption of Kolibri. 
+
+In the early days of `kUSD` there was nothing to do with the asset besides sell it for `XTZ`. This effectively meant `kUSD` minters could take long positions on the price of `XTZ`. The other use case available to minters were peer to peer transactions of `kUSD` and holding `kUSD` to liquidate ovens. Demand for long positions greatly outstripped other use cases, and [KIP006](https://forum.tezosagora.org/t/kolibri-governance-proposal-6-continuous-increase-of-the-stability-fee-to-reach-peg/2851) continuously raised the stability fee until peg was achieved. 
```

Comment:
I'm not sure I'd refer to that former gov proposal as KIP006, that's pretty confusing IMO.

Reply:
Renamed to "our initial fee raising proposal" but I don't love it. Happy to take a name suggestion.